src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -3,7 +3,7 @@
 @include pf-root($code-editor) {
   // controls
   --#{$code-editor}__controls--PaddingInlineStart: var(--pf-t--global--spacer--sm);
-  --#{$code-editor}__controls--Gap: var(--pf-t--global--spacer--gap--control-to-control--plain);
+  --#{$code-editor}__controls--Gap: var(--pf-t--global--spacer--gap--action-to-action--plain);
```

Comment:
Updated to match the suggestion in https://github.com/patternfly/patternfly/pull/6957#discussion_r1747657764. Looks like a typo, `--pf-t--global--spacer--gap--control-to-control--plain` isn't a valid token.